tensorflow/java/BUILD

```diff
@@ -40,6 +41,31 @@ filegroup(
     ],
 )
 
+java_plugin(
+    name = "operator_plugin",
+    generates_api = True,
+    processor_class = "org.tensorflow.processor.operator.OperatorProcessor",
+    visibility = ["//visibility:public"],
+    deps = [":operator_plugin_library"],
+)
+
+java_library(
+    name = "operator_plugin_library",
+    srcs = [":operator_plugin_sources"],
+    javacopts = JAVACOPTS,
+    resources = [":operator_plugin_config"],
```

Comment:
Same here - inline the resource instead of indirecting through a `filegroup`?

Reply:
done